diff --git a/setup.py b/setup.py
index 7a601d5..59e3306 100644
--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,7 @@ setup(
     author='Linden Lab',
     author_email='eventletdev@lists.secondlife.com',
     url='http://eventlet.net',
-    packages=find_packages(exclude=['tests', 'benchmarks']),
+    packages=find_packages(exclude=['tests', 'tests.manual', 'benchmarks']),
     install_requires=(
         'greenlet >= 0.3',
     ),
